### ISSUE TITLE
Timer continues counting after level has been completed, and when the grid is obscured

### ISSUE THREAD

**Issue by thurstylark** [jaredforrest/nonocross#38]
If the user intends to dismiss the level complete prompt to continue playing a generated level that has more than one solution, the time spent in the completion prompt still counts towards the Time Taken.

For a level that has more than one solution, it makes sense to continue counting again when the level complete prompt has been dismissed, as the time that the user is considering the completed puzzle should count towards the total Time Taken. However, if the level has only one solution, the timer should not be counting whenever the level is in a completed state.

To put it another way, the timer should only be counting when:
1. The user is able to view the whole board (as in, not obstructed by a prompt, QR code, etc), or
2. There are no more moves that the user can make to complete the level (*except* in the case of a level that has more than one solution)
